docs/source/actions/copy.rst

```diff
@@ -42,10 +42,14 @@ Options
      - ``true``
      - Whether to create the pull requests even if they are conflicts when
        cherry-picking the commits.
+   * - ``labels``
+     - list of string
+     - ``[]``
+     - The list of labels to add to the created pull requests.
    * - ``label_conflicts``
      - string
      - ``conflicts``
-     - The label to add to the created pull requested if it has conflicts and
+     - The label to add to the created pull requests if it has conflicts and
```

Comment:
Thanks, I fixed a bunch of those in https://github.com/Mergifyio/mergify-engine/pull/2335 !